### PR TITLE
Speakers, not Papers.

### PR DESCRIPTION
**Pull request by milessabin**

### CODE REVIEW

File: _events/2016-03-summit-philadelphia.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ If you're interested in types and pure functional programming we'd love to see y
 
 <a class="btn large" href="https://www.eventbrite.co.uk/e/typelevel-summit-us-tickets-20778897241">Buy tickets</a>
 
-## Call for Papers
+## Call for Speakers
 
 The Summit will feature both 30 minute and 15 minute-long talks.
 The Call for Papers is joint with the <a href="/event/2016-05-summit-oslo/">Oslo Summit</a>.
```
Comment (adelbertc) [inline _events/2016-03-summit-philadelphia.md]: Need to fix here too

File: _events/2016-05-summit-oslo.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ sponsors_section: true
 
 The second Typelevel Summit will be co-located with <a href="http://2016.flatmap.no/">flatMap(Oslo)</a>.
 
-## Call for Papers
+## Call for Speakers
 
 The Summit will feature both 30 minute and 15 minute-long talks.
 The Call for Papers is joint with the <a href="/event/2016-03-summit-philadelphia/">Philadelphia Summit</a>.
```
Comment (adelbertc) [inline _events/2016-05-summit-oslo.md]: and here (and maybe other places not viewable)

**Comment by adelbertc** [comment]
:+1:

**Comment by ceedubs** [comment]
:+1:

**Comment by ceedubs** [comment]
I was planning on merging, but I don't have rights.

**Pull request merged**